pbin.sh

```diff
@@ -100,13 +106,13 @@ set_defaults() {
 	unset reply
 
 	# default to user@hostname
-	name=${SUDO_USER:-$USER}@${HOSTNAME:-$(hostname)}
+	name=${SUDO_USER:-$USER}
```

Comment:
You won't see them in the list of recent or trending though as secure pastes don't show there.

So you'd need the link in the first (unless you fish from the DB) so at that point you'd know who they were from hopefully. If it's a public paste then I think we shouldn't put this info (security concern?) up but that's just imho and I'm not overly precious about it.